src/GitHook/Command/FileCommand/PreCommit/PhpMd/PhpMdCheckConfiguration.php

```diff
@@ -34,4 +36,34 @@ public function getMinimumPriority(): int
 
         return (isset($this->config['priority'])) ? $this->config['priority'] : $defaultPriority;
     }
+
+    /**
+     * @return string
+     */
+    public function getRulesetPath(): string
+    {
+        return $this->config['rulesetPath'] ?? $this->getDefaultRulesetPath();
+    }
+
+    /**
+     * @throws \Exception
+     *
+     * @return string
+     */
+    protected function getDefaultRulesetPath(): string
+    {
+        $defaultRuleSetPathsByPriority = [
+            'config/phpmd-ruleset.xml',
+            'vendor/spryker/spryker/Bundles/Development/src/Spryker/Zed/Development/Business/PhpMd/ruleset.xml',
+            'vendor/spryker/development/src/Spryker/Zed/Development/Business/PhpMd/ruleset.xml',
```

Comment:
Core usually has the config for only the split scenario. You can have a look at how the `DevelopmentConfig::INTERNAL_NAMESPACES_TO_PATH_MAPPING` is defined and overridden on the project level.

The comment `@project Only needed in Project, not in demoshop` is required in this case.